contracts/farm/LiquidityMiningV2.sol

```diff
@@ -552,7 +561,7 @@ contract LiquidityMiningV2 is ILiquidityMiningV2, LiquidityMiningStorageV2 {
 	 * @param _poolToken the address of pool token
 	 * @param _user the address of user to claim reward from (can be passed only by wrapper contract)
 	 */
-	function claimRewards(address _poolToken, address _user) external {
+	function claimRewards(address _poolToken, address _user) external onlyAfterMigrationFinished {
```

Comment:
I mean, this is fine but unless we have finished and moved all the funds this will not do anything, I guess.